src/dockerflow/sanic/checks.py

```diff
@@ -5,21 +5,7 @@
 This module contains built-in checks for the Sanic integration.
 """
 from .. import health
-from ..checks import (  # noqa
-    CRITICAL,
-    DEBUG,
-    ERROR,
-    INFO,
-    STATUSES,
-    WARNING,
-    CheckMessage,
-    Critical,
-    Debug,
-    Error,
-    Info,
-    Warning,
-    level_to_text,
-)
```

Comment:
Wouldn't this be a breaking change?

Reply:
Ahh yeah, probably best to keep this in place.